modul/_version.py

```
@@ -6,30 +6,37 @@
+VERSION: str
 __version__: str
-__version_tuple__: VERSION_TUPLE
-version_tuple: VERSION_TUPLE
-commit_id: COMMIT_ID
-__commit_id__: COMMIT_ID
+__version_tuple__: VersionTuple
+version_tuple: VersionTuple
+COMMIT_ID: CommitId
+__commit_id__: CommitId
 
+<<<<<<< HEAD
```

Pylint (4.0.4)

[error] Parsing failed: 'invalid syntax (modul._version, line 32)'

(E0001)